test/Mono.Linker.Tests.Cases/RequiresCapability/RequiresUnreferencedCodeCapability.cs

```diff
@@ -24,11 +28,20 @@ public static void Main ()
 			TestRequiresSuppressesWarningsFromReflectionAnalysis ();
 			TestDuplicateRequiresAttribute ();
 			TestRequiresUnreferencedCodeOnlyThroughReflection ();
+			TestBaseTypeVirtualMethodRequiresUnreferencedCode ();
+			TestTypeWhichOverridesMethodVirtualMethodRequiresUnreferencedCode ();
+			TestTypeWhichOverridesMethodVirtualMethodRequiresUnreferencedCodeOnBase ();
+			TestStaticCctorRequiresUnreferencedCode ();
+			TestDynamicallyAccessedMembersWithRequiresUnreferencedCode (typeof (DynamicallyAccessedTypeWithRequiresUnreferencedCode));
+			TestInterfaceMethodWithRequiresUnreferencedCode ();
+			TestCovariantReturnCallOnDerived ();
+			TestRequiresInMethodFromCopiedAssembly ();
+			TestRequiresThroughReflectionInMethodFromCopiedAssembly ();
 		}
 
 		[ExpectedWarning ("IL2026",
-			"Calling 'Mono.Linker.Tests.Cases.RequiresCapability.RequiresUnreferencedCodeCapability.RequiresWithMessageOnly()' " +
-			"which has `RequiresUnreferencedCodeAttribute` can break functionality when trimming application code. " +
+			"'Mono.Linker.Tests.Cases.RequiresCapability.RequiresUnreferencedCodeCapability.RequiresWithMessageOnly()' method " +
+			"has 'RequiresUnreferencedCodeAttribute' which can break functionality when trimming application code. " +
```

Comment:
Since you're here - can you just remove the full text validation - we should validate only the warning code and than the presence of the value parameters for the message - not the message itself.